src/contentsharecontroller/DefaultContentShareController.ts

```diff
@@ -48,10 +50,21 @@ export default class DefaultContentShareController
         this.stopContentShare();
       });
     }
-    this.audioVideo.start();
+    this.contentAudioVideo.start();
     if (this.mediaStreamBroker.mediaStream.getVideoTracks().length > 0) {
-      this.audioVideo.videoTileController.startLocalVideoTile();
+      this.contentAudioVideo.videoTileController.startLocalVideoTile();
     }
+
+    const tile = this.attendeeAudioVideo.videoTileController.addVideoTile();
+    tile.bindVideoStream(
+      this.contentAudioVideo.configuration.credentials.attendeeId,
+      false,
+      stream,
+      null,
```

Comment:
can we fill the width and height ?

Reply:
We do not set width and height for content share currently. I checked the current behavior and it does not contain width and height either.